tap_lightspeed/client.py

```diff
@@ -115,7 +116,9 @@ def request_decorator(self, func: Callable) -> Callable:
             backoff.expo,
             (
                 RetriableAPIError,
-                TooManyRequestsError
+                TooManyRequestsError,
+                ImproperConnectionState,
+                ConnectionError,
```

Comment:
Don't we need to import `ConnectionError`?

Reply:
no, it is a standard python error